src/auditor/compiler_abi.jl

```diff
@@ -185,7 +185,7 @@ function check_cxxstring_abi(oh::ObjectHandle, platform::Platform; io::IO = stdo
         both GCC 4 and GCC 5.  To do this, immediately after your `products`
         definition in your `build_tarballs.jl` file, add the line:
         """, '\n' => ' '))
-        msg *= "\n\n    products = expand_cxx_versions(products)"
+        msg *= "\n\n    platforms = expand_cxxstring_abis(platform)"
```

Comment:
Sorry, I'm apparently unable to type:
```suggestion
        msg *= "\n\n    platforms = expand_cxxstring_abis(platforms)"
```
Also the rest of the message needs to be adapted as it mentions products